modules/network-observability-multitenancy.adoc

```diff
@@ -22,3 +24,10 @@ $ oc adm policy add-cluster-role-to-user netobserv-reader user1
 +
 Now, the data is restricted to only allowed user namespaces. For example, a user that has access to a single namespace can see all the flows internal to this namespace, as well as flows going from and to this namespace.
 Project admins have access to the Administrator perspective in the {product-title} console to access the Network Flows Traffic page.
+
+* To enable multi-tenancy for developer perspective, run the following command:
++
+[source,terminal]
+----
+$ oc adm policy add-role-to-user netobserv-metrics-reader <user_group_or_name> -n <your_namespace>
+----
```

Comment:
@jotak @jpinsonneau Ok, how does this look? I'm unsure if I should add it here or not, since technically this section is nested under the Loki stuff. I could bring it out from underneath the Loki stuff though, since it applies to non-Loki use cases as well. 

Is there anything else we need to say besides MT is available for Loki and Prometheus in Developer perspective but needs this access? If this is OK, I will link it to the Developer perspective info in the Overview too.

Reply:
Can't we get this section right after `Installing the Network Observability Operator`, at the same level ?

It will make more sense as we need to install the operator to get the roles created.
Then we can have 2 optional cases for Loki and Prometheus.

Reply:
+1 @jpinsonneau 
By the way, we should also mention the `cluster-monitoring-view` role I guess, for the admin perspective? Users might very well already have it because it's not specific to netobserv, but a project admin who uses the administrator perspective needs that role too:

```bash
$ oc adm policy add-cluster-role-to-user cluster-monitoring-view <user_group_or_name>
```

Reply:
@jotak so for the admin perspective, the command would be this?:
`$ oc adm policy add-cluster-role-to-user cluster-monitoring-view netobserv-reader <user_group_or_name>`

Or should I separate the `cluster-monitoring-view`?